projects/pnpm.io/package.yml

```diff
@@ -0,0 +1,26 @@
+distributable:
+  url: https://registry.npmjs.org/pnpm/-/pnpm-{{version}}.tgz
+  strip-components: 1
+
+versions:
+  github: pnpm/pnpm
+
+provides:
+  - bin/pnpm.cjs
+  - bin/pnpx.cjs
+
+dependencies:
+  nodejs.org: '>=16'
+  npmjs.com: '>=8'
+
+build:
+  script: |
+    npm install
+    export PNPM_HOME=root/Library/pnpm
+    export PATH="$PNPM_HOME:$PATH";
+    export PATH="$(pwd)/bin:$PATH";
```

Comment:
your script doesn't move anything to `{{prefix}}` so nothing is installed. Confused what you're doing anyway, all brew does is move a few files: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pnpm.rb#L29-L33

Reply:
sorry did not fully understand the purpose of {{prefix}}, i just wrongly assumed pwd would be where to just put stuff. but thanks im finally getting a different error now.